### PR TITLE
Rename DPC++ to SYCL in metadata and comments

### PR DESCRIPTION
**Pull request by KanclerzPiotr**
# Existing Sample Changes
## Description

Rename DPC++ to SYCL in metadata and comments 

Fixes Issue# 

## Type of change

- [X] Implement fixes for ONSAM Jiras

## How Has This Been Tested?

- [X] Command Line

### CODE REVIEW

File: DirectProgramming/DPC++/Jupyter/oneapi-essentials-training/12_DPCPP_Local_Memory_And_Atomics/run_matrixmul_16x16_localmem.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 source /opt/intel/oneapi/setvars.sh > /dev/null 2>&1
-/bin/echo "##" $(whoami) is compiling DPCPP_Essentials Module12 -- DPCPP Atomics Local Memory - 5 of 5 matrixmul_16x16_localmem.cpp
+/bin/echo "##" $(whoami) is compiling SYCL_Essentials Module12 -- SYCL tomics Local Memory - 5 of 5 matrixmul_16x16_localmem.cpp
```
Comment (jimmytwei) [inline DirectProgramming/DPC++/Jupyter/oneapi-essentials-training/12_DPCPP_Local_Memory_And_Atomics/run_matrixmul_16x16_localmem.sh:3]: @KanclerzPiotr Change "SYLC tomics" to "SYCL Atomics"

File: DirectProgramming/DPC++/Jupyter/sycl-performance-portability-training/00_Introduction.ipynb [hunk 2]
```diff
@@ -388,7 +388,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.10"
+   "version": "3.9.7"
+  },
+  "vscode": {
```
Comment (jimmytwei) [inline DirectProgramming/DPC++/Jupyter/sycl-performance-portability-training/00_Introduction.ipynb:393]: @KanclerzPiotr Can you remove vscode?

File: DirectProgramming/DPC++/Jupyter/oneapi-essentials-training/12_DPCPP_Local_Memory_And_Atomics/run_localmem_info.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 source /opt/intel/oneapi/setvars.sh > /dev/null 2>&1
-/bin/echo "##" $(whoami) is compiling DPCPP_Essentials Module12 -- DPCPP Atomics Local Memory - 3 of 5 localmem_info.cpp
+/bin/echo "##" $(whoami) is compiling SYCL_Essentials Module12 -- SYCL tomics Local Memory - 3 of 5 localmem_info.cpp
```
[the remark this answers is not in the document]
Comment (KanclerzPiotr) [reply DirectProgramming/DPC++/Jupyter/oneapi-essentials-training/12_DPCPP_Local_Memory_And_Atomics/run_localmem_info.sh]: Fixed

File: DirectProgramming/DPC++/Jupyter/sycl-performance-portability-training/00_Introduction.ipynb (hunk 2 above)
Comment (KanclerzPiotr) [reply DirectProgramming/DPC++/Jupyter/sycl-performance-portability-training/00_Introduction.ipynb]: Removed the metadata changes with vscode

**Review by jimmytwei — APPROVED** [review]

**Pull request merged**